### ISSUE TITLE
[Bug]: export onnx fail

### ISSUE THREAD

**Issue by dugushiyu** [openvinotoolkit/anomalib#1388]
### Describe the bug

export.py 
export 
trainer.datamodule.test_data.transform.to_dict()        fail
'compose' object  has no attribute ' to_dict'

### Dataset

Folder

### Model

PADiM

### Steps to reproduce the behavior

python tools/train.py --model default

### OS information

OS information:
- OS: [e.g. windows]
- Python version: [e.g. 3.8.8]
- Anomalib version: [e.g. 0.7.0]
- PyTorch version: [e.g. 1.12.0]
- Any other relevant information: [e.g. I'm using a custom dataset]

### Expected behavior

onnx success

### Pip/GitHub

pip

### What version/branch did you use?

main

### Configuration YAML

```yaml
train success 
optimization:
export_mode:onnx
```

### Logs

```shell
loop.py line207 in run   output=self.on_run_end()
fit_loop.py line 325 in on_run_end    self.trainer.__callback__hooks("on_train_end")
trainer.py line 1596  in _call_callback_hooks  fn(*****)
export.py line51  in on_train_end  transform=trainer.datamodelu.test_data.transform.to_dict()

attributeerror:'Compose'object has no attribute 'to_dict'
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blaz-r** [comment]
Hello. Could you please provide the entire configuration file, and since the error comes from transformations, could you also provide that information please.

**Comment by samet-akcay** [comment]
I think this is related to the albumentations version. Which albumentation version do you use?

**Comment by dugushiyu** [comment]
> I think this is related to the albumentations version. Which albumentation version do you use?

albumentation 1.2.0

**Comment by dugushiyu** [comment]
> Hello. Could you please provide the entire configuration file, and since the error comes from transformations, could you also provide that information please.

dataset:
  name: tube               
  format: folder
  path: ./datasets/img_192 
  normal_dir: normal       
  abnormal_dir: null  
  mask_dir: null          
  normal_test_dir: null # name of the folder containing normal test images.
  task: classification # classification or segmentation
  extensions: null
  normalization: imagenet 
  split_ratio: 0.2 # ratio of the normal images that will be used to create a test split
  image_size: 256
  train_batch_size: 32
  test_batch_size: 32
  num_workers: 8
  transform_config:
    train: null
    val: null
  test_split_mode: from_dir 
  test_split_ratio: 0.2
  val_split_mode: same_as_test
  val_split_ratio: 0.5
  create_validation_set: true
  tiling:
    apply: false
    tile_size: null
    stride: null
    remove_border_count: 0
    use_random_tiling: False
    random_tile_count: 16
	
.
.
.
.
.

optimization:
  export_mode: onnx # options: torch, onnx, openvino
  
.
.
.
.

**Comment by samet-akcay** [comment]
Closing this as albumentation support is now removed.
